Makefile

```
@@ -1,58 +1,43 @@
 PROJ_BASE=$(shell pwd)
-PYTHONVER=python3.9
 PYTHONVENV=$(PROJ_BASE)/venv
-VENVPYTHON=$(PYTHONVENV)/bin/$(PYTHONVER)
-
-install: bootstrap
-	@echo "Installing WKTPlot"
-	$(VENVPYTHON) setup.py install
-	@echo "\nYou may want to activate the virtual environmnent with 'source venv/bin/activate'\n"
-
-develop: bootstrap
-	@echo "Installing WKTPlot, with editible modules ('python setup.py develop')"
-	$(VENVPYTHON) setup.py develop
-	@echo "\nYou may want to activate the virtual environmnent with 'source venv/bin/activate'\n"
-
-bootstrap:
-	@echo "Creating virtual environment 'venv' for development."
-	$(PYTHONVER) -m virtualenv -p $(PYTHONVER) venv
-	@echo "Installing python modules from requirements.txt"
-	$(VENVPYTHON) -m pip install -r requirements.txt
-
-clean_build:
+ifeq ($(OS),Windows_NT)
+	VENVPYTHON=$(PYTHONVENV)/Scripts/python
+else
+	VENVPYTHON=$(PYTHONVENV)/bin/python
+endif
+
+PHONY: init
+init:
+	@echo "Creating virtual environment 'venv'."
+	python3 -m venv venv || python -m venv venv
+	$(VENVPYTHON) -m pip install --upgrade setuptools pip
```

```suggestion
	$(VENVPYTHON) -m pip install --upgrade setuptools pip pip-tools
```

I'm suggesting adding `pip-tools` to this step so developers can generate  [requirements.txt](https://github.com/FuzzFoundation/WKTPlot/blob/refactor/upgrade-to-use-setup-cfg/requirements.txt#L1-L6)